fabtests/pytest/common.py

```diff
@@ -173,7 +230,18 @@ def __init__(self, cmdline_args, base_command, is_negative=False, failing_warn_m
         self._failing_warn_msgs = failing_warn_msgs
         self._base_command = base_command
         self._is_negative = is_negative
-        self._command = self._cmdline_args.populate_command(base_command, "host")
+        if "neuron" in base_command and "PYTEST_XDIST_WORKER" in os.environ:
+            host_ip = self._cmdline_args.server_id
+            worker_id = int(os.environ["PYTEST_XDIST_WORKER"].replace("gw", ""))
+            neuron_device_id = worker_id % num_neuron_devices(host_ip)
+            num_cores = num_neuron_cores_on_device(host_ip, neuron_device_id)
+            additional_environment = "NEURON_RT_VISIBLE_CORES={}".format(
```

Comment:
Question. Is it possible to run multiple processes on different cores on the the same Neuron device?

Reply:
Yes. that is possible.